action/well_known.go

```diff
@@ -0,0 +1,66 @@
+package action
+
+import (
+	"github.com/deislabs/cnab-go/claim"
+	"github.com/deislabs/cnab-go/credentials"
+	"github.com/deislabs/cnab-go/driver"
+)
+
+// Well known constants define the Well Know CNAB action to be taken
+const (
+	ActionDryRun     = "io.cnab.dry-run"
+	ActionHelp       = "io.cnab.help"
+	ActionLog        = "io.cnab.log"
+	ActionStatus     = "io.cnab.status"
+	ActionStatusJSON = "io.cnab.status+json"
+)
+
+// DryRun runs a dry-run action on a CNAB bundle.
+type DryRun struct {
+	Driver driver.Driver
+}
+
+// Help runs a help action on a CNAB bundle.
+type Help struct {
+	Driver driver.Driver
+}
+
+// Log runs a log action on a CNAB bundle.
+type Log struct {
+	Driver driver.Driver
+}
+
+// Status runs a status action on a CNAB bundle.
+type Status struct {
+	Driver driver.Driver
+}
+
+// StatusJSON runs a status+JSON action on a CNAB bundle.
+type StatusJSON struct {
+	Driver driver.Driver
+}
+
+// Run executes a dry-run action in an image
+func (i *DryRun) Run(c *claim.Claim, creds credentials.Set, opCfgs ...OperationConfigFunc) error {
+	return (&RunCustom{Driver: i.Driver, Action: ActionDryRun}).Run(c, creds, opCfgs...)
+}
+
+// Run executes a help action in an image
+func (i *Help) Run(c *claim.Claim, creds credentials.Set, opCfgs ...OperationConfigFunc) error {
+	return (&RunCustom{Driver: i.Driver, Action: ActionHelp}).Run(c, creds, opCfgs...)
+}
+
+// Run executes a status action in an image
+func (i *Log) Run(c *claim.Claim, creds credentials.Set, opCfgs ...OperationConfigFunc) error {
+	return (&RunCustom{Driver: i.Driver, Action: ActionLog}).Run(c, creds, opCfgs...)
+}
+
+// Run executes a status action in an image
+func (i *Status) Run(c *claim.Claim, creds credentials.Set, opCfgs ...OperationConfigFunc) error {
+	return (&RunCustom{Driver: i.Driver, Action: ActionStatus}).Run(c, creds, opCfgs...)
+}
+
+// Run executes a status+JSON action in an image
+func (i *StatusJSON) Run(c *claim.Claim, creds credentials.Set, opCfgs ...OperationConfigFunc) error {
+	return (&RunCustom{Driver: i.Driver, Action: ActionStatusJSON}).Run(c, creds, opCfgs...)
+}
```

Comment:
I see that this is already in the [spec](https://github.com/deislabs/cnab-spec/blob/master/804-well-known-custom-actions.md), so it may be outside the scope of discussion for this PR. But it seems like there should be no need for a separate `io.cnab.status+json` action when `io.cnab.status` also exists given that `io.cnab.status` could define a [cnab output](https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md#outputs) for the `status.json` file.

Reply:
Started discussion in https://github.com/deislabs/cnab-spec/pull/290

Reply:
Update: the spec change is merged, so let's remove this.